saylua/static-source/scss/layout/_navbar.scss

```diff
@@ -185,9 +206,97 @@ $navHeight: 4rem;
   display: none;
 }
 
-// Prevent the navbar from being sticky on mobile because that's annoying. 
-@media (max-width: 1024px) {
+.navbar-dropdown-main .navbar-link i {
```

Comment:
Hmm that's true. I'm using them as font awesome icons but yeah, I guess it would do that too. Unless we used <em>